projects/step-frontend/src/lib/modules/execution/components/schedule-overview/scheduler-page.component.html

```diff
@@ -0,0 +1,63 @@
+<section class="header" [class.small-screen]="!!(_isSmallScreen$ | async)">
+  @if (task(); as task) {
+    <div class="title">
+      <h3 data-step-testid>
+        <entity-icon entityName="tasks" [entity]="task"></entity-icon>
+
+        <div>{{ task.attributes?.['name'] }}</div>
+        @if ((lastExecution$ | async)?.execution; as lastExecution) {
+          <a
+            class="plan-link"
+            routerLink="/plans/editor/{{ lastExecution.planId }}"
```

Comment:
I would prefer something like: 
[routerLink]="['/plans/editor', lastExecution.planId]"

(please test it before doing the change)